components/Container.js

```diff
@@ -45,14 +45,14 @@ export default function Container(props) {
           <meta property="article:published_time" content={meta.date} />
         )}
       </Head>
-      <nav className="sticky-nav flex justify-between items-center max-w-4xl w-full p-8 my-0 md:my-8 mx-auto bg-white dark:bg-black bg-opacity-60">
-        <a href="#skip" className="sr-only focus:not-sr-only">
+      <nav className="flex items-center justify-between w-full max-w-4xl p-8 mx-auto my-0 text-gray-900 bg-white sticky-nav md:my-8 dark:bg-black bg-opacity-60 dark:text-gray-100">
+        <a href="#skip" className="skip-nav">
           Skip to content
         </a>
         <button
           aria-label="Toggle Dark Mode"
           type="button"
-          className="bg-gray-200 dark:bg-gray-800 rounded p-3 h-10 w-10"
+          className="w-10 h-10 p-3 bg-gray-200 rounded dark:bg-gray-800"
```

Comment:
Why did these change?

Reply:
I use the [headwind](https://github.com/ryanhhhh/headwind) extension and it sorted the classes.